From 5533ae323d186df61d775b5bcfdb9f02dafd5f99 Mon Sep 17 00:00:00 2001
From: Michael Gratton <mike@vee.net>
Date: Sun, 25 Oct 2020 19:00:36 +1100
Subject: [PATCH 116/124] Composer.Widget: Hide save button by default

Determining if this can be enabled currently requires opening the
remote, so hide it by default until we know for sure.
---
 src/client/composer/composer-widget.vala | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/client/composer/composer-widget.vala b/src/client/composer/composer-widget.vala
index 46986b8b..6b158f83 100644
--- a/src/client/composer/composer-widget.vala
+++ b/src/client/composer/composer-widget.vala
@@ -439,6 +439,8 @@ public class Composer.Widget : Gtk.EventBox, Geary.BaseInterface {
 
         this.header = new Headerbar(config);
         this.header.expand_composer.connect(on_expand_compact_headers);
+        // Hide until we know we can save drafts
+        this.header.show_save_and_close = false;
 
         // Setup drag 'n drop
         const Gtk.TargetEntry[] target_entries = { { URI_LIST_MIME_TYPE, 0, 0 } };
-- 
2.29.2

